### PR TITLE
Fix.reading time

### LINKED ISSUE #398: iojs will be mandatory
I am going to use jsdom for fast module testing during dev and latest version doesn't work with node anymore.
I think this situation is just a start, so we are going to force iojs.
Are you all ok with that ?
(Note: I recommand nvm)

### PR DESCRIPTION
**Pull request by MoOx**
Like we do for work with @bloodyowl, I used jsdom for quick testing during dev + testem for full local testing for "npm test".

@bloodyowl I use a simpler method: jsdom console output directly forwarded: less code and you get ALL errors from browser without having to open by hand the url (eg: in my case I directly see that classList wasn't supported by jsdom yet).

iojs is now mandatory (this close #398)

Build is probably broken for now, will fix later.

### CODE REVIEW

**Comment by MoOx** [comment]
Fixed

**Comment by MoOx** [comment]
up

**Comment by bloodyowl** [comment]
LGTM, although I don't think we ~really~ need to use cornea for that.

**Comment by MoOx** [comment]
I just used your previous code. I agree with that but I didn't rewrite the feature I just put it back :)

**Comment by MoOx** [comment]
Maybe I can get rid of it and just use simple & stupid plain js ?

**Comment by bloodyowl** [comment]
given how simple it is, you can, yeah :)

**Comment by bloodyowl** [comment]
we can even pre-render it, right?

**Comment by MoOx** [comment]
we might be able indeed by stripping html tag in markdown htmlized metalsmith content. I will try that.

File: contributors.json [hunk 1]
```diff
@@ -110,6 +110,15 @@
       "location": "Aix-en-Provence",
       "hireable": true
     },
+    "dguyon@gmail.com": {
```
Comment (Nyalab) [inline contributors.json]: careful, don't add him back

Comment (MoOx) [inline contributors.json]: He made some commits so yes it should be here. We should just had removed his .json file.

File: src/modules/reading-time/index.es [hunk 2]
```diff
@@ -0,0 +1,45 @@
+import cornea from "cornea"
+
+export default cornea.extend({
```
Comment (Nyalab) [inline src/modules/reading-time/index.es]: why no ES6?

Comment (MoOx) [inline src/modules/reading-time/index.es]: I just quickly adapted old code. Anyway, like said in the PR below, I am going to make this "server" side.

File: contributors.json (hunk 1 above)
Comment (Nyalab) [inline contributors.json]: ok thx

File: src/modules/reading-time/index.es (hunk 2 above)
Comment (Nyalab) [inline src/modules/reading-time/index.es]: :+1:

**Comment by Nyalab** [comment]
lgtm, i just don't get why you need to add a polyfill for classList, but i didn't dig to find why.

anyway, nice work, tell me if you need a review when you will have refactored it

**Comment by MoOx** [comment]
polyfill for classList is for jsdom (it doesn't support classList yet) https://github.com/tmpvar/jsdom/issues/510

**Comment by MoOx** [comment]
Ok I've updated that, we now have `<ReadingTime />` as a react component :)

**Comment by MoOx** [comment]
travis now run some tests on ff and phantomjs 

```console
$ npm test

> putaindecode.fr@0.0.0 test /home/travis/build/putaindecode/putaindecode.fr
> npm run lint && npm run tape && npm run build && npm run testem


> putaindecode.fr@0.0.0 lint /home/travis/build/putaindecode/putaindecode.fr
> eslint -f node_modules/eslint-friendly-formatter --ext .js,.jsx,.es .


> putaindecode.fr@0.0.0 tape /home/travis/build/putaindecode/putaindecode.fr
> babel-tape-runner 'scripts/**/__tests__/*'


> putaindecode.fr@0.0.0 build /home/travis/build/putaindecode/putaindecode.fr
> __VERSION__=$(git log -n 1 --pretty=format:'%h') babel-node scripts/build

[07:54:16] [contributors] ✓ Contributors cache updated
[07:54:16] [contributors] ✓ Top contributions done
[07:54:20] [contributors] ✓ Contributions per files done

✓ Static build completed

✓ Assets build completed

> putaindecode.fr@0.0.0 testem /home/travis/build/putaindecode/putaindecode.fr
> testem ci -f scripts/testem.conf.yml

ok 1 Firefox 31.0 - should say when you have less than a minute
ok 2 Firefox 31.0 - should offer a tooltip as attribute
ok 3 Firefox 31.0 - should use cssrecipes-tooltip
ok 4 Firefox 31.0 - should say when you have more than one minute
ok 5 Firefox 31.0 - should say when you have 10 minutes
ok 6 PhantomJS 1.9 - should say when you have less than a minute
ok 7 PhantomJS 1.9 - should offer a tooltip as attribute
ok 8 PhantomJS 1.9 - should use cssrecipes-tooltip
ok 9 PhantomJS 1.9 - should say when you have more than one minute
ok 10 PhantomJS 1.9 - should say when you have 10 minutes

1..10
# tests 10
# pass  10
# fail  0

# ok

The command "npm test" exited with 0.

Done. Your build exited with 0.
```

**Comment by lionelB** [comment]
nice nice nice!!!

**Comment by MoOx** [comment]
C'est que le début :)

**Pull request merged**